Documentation/Contributing.md

```diff
@@ -9,89 +9,48 @@ Root
  |
  |- Workbooks
        |- Category A
-             |- categoryResources.json
              |- Template A
                     |- TemplateA.workbook
-                    |- settings.json
                     |- icon.svg
              |- Template B
                     |- TemplateB.workbook
-                    |- settings.json
                     |- icon.svg
        |- Category B
-             |- categoryResources.json
              |- Template C
                     |- TemplateC.workbook
-                    |- settings.json
                     |- icon.svg
              |- Template D
                     |- TemplateD.workbook
-                    |- settings.json
                     |- icon.svg
 ```
-## Category folder
+
+## Gallery folder
 ```
 Root
  |
- |- Workbooks
-       |- Category folder 1
-       |- Category folder 2       
+ |- gallery folder
```

Comment:
>gallery [](start = 4, length = 7)

do you want this folder to be one level down inside workbooks?  to move the cohorts specific stuff into the cohorts folder, workbook specific ones stay inside workbooks?

Reply:
I prefer the gallery folder to be a separate thing than a subfolder within Workbooks and Cohorts, it makes the build cleaner and less complicated. But I think I will create subfolders for the gallery for workbook type and then the gallery name just to be the workbook type.